config/samples/cc-demo-runtime.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: confidentialcontainers.org/v1beta1
+kind: CcRuntime
+metadata:
+  name: ccruntime-demo
```

Comment:
how about we make the name explicit: `ccruntime-ssh-demo` ?

Reply:
And a super nit :-) 
Ok to name the file as `ccruntime-ssh-demo.yaml` ?

Reply:
Thanks for the comments. I've made the updates